game/src/test/scala/io/aigar/game/CellSpec.scala

```diff
@@ -224,4 +227,13 @@ class CellSpec extends FlatSpec with Matchers {
 
     opponent.cells should contain only largeCell
   }
+
+  "performAction" should "change target with the right action" in {
```

Comment:
I see why you propose that, but I don't feel bad about an action having multiple effects. But w/e